src/main/java/com/gluonhq/richtext/model/TextDecoration.java

```diff
@@ -67,6 +67,21 @@ public TextDecoration build() {
             return decoration;
         }
 
+        /**
+         * Returns a new instance of builder by copying the properties from the supplied decoration
+         * @param decoration TextDecoration to copy properties from
+         * @return An instance of TextDecoration.Builder
+         */
+        public static Builder copy(TextDecoration decoration) {
```

Comment:
Nitpicking... this is not `copy`,  since we are creating `builder` from `decoration`.
May be it should be called `builder`, so we have static and dynamic builder methods?

Reply:
I have moved this method into TextDecoration and now it doesn't need an instance of TextDecoration as a parameter. I cannot rename it to `builder()` as the method definition is same. I do not have a better name so I have left it as `copy()` for now.

Reply:
Nice! What about `asBuilder` or `toBuilder` then?

Reply:
`toBuilder` sounds good to me. Renamed!